### PR TITLE
feat: Added Vanuatu Vatu to currency list

### PR DESCRIPTION
**Pull request by nzidol**
## Description

> Added missing currency for Vanuatu

## Type of change

- New feature (non-breaking change which adds functionality)
- Addition to list of constants

### CODE REVIEW

File: app/client/src/constants/Currency.tsx [hunk 1]
```diff
@@ -1546,6 +1546,14 @@ export const CurrencyTypeOptions: Array<CurrencyOptionProps> = [
...
     symbol_native: "₫",
   },
+  {
+    code: "VUV",
+    currency: "VT",
+    currency_name: "Vatu",
+    label: "Vanuatu",
+    phone: "678",
+    symbol_native: "VT",
+  },
```
Comment (Tooluloope) [inline app/client/src/constants/Currency.tsx:1556]:
```
{
    code: "VU",
    currency: "VUV",
    currency_name: "Vanuatu Vatu",
    label: "Vanuatu",
    phone: "678",
    symbol_native: "VT",
  }
  ```
  
  Hi @nzidol 
  Thanks for making this contribution, Here is the correct detail for Vanuatu

**Review by Tooluloope — CHANGES_REQUESTED** [review]

File: app/client/src/constants/Currency.tsx (hunk 1 above)
Comment (nzidol) [reply app/client/src/constants/Currency.tsx:1556]: updated

**Review by Tooluloope — APPROVED** [review]

**Comment by Tooluloope** [comment]
@mohanarpit can you help merge this PR

**Comment by mohanarpit** [comment]
@nzidol Thank you so much for your contribution towards making Appsmith a better product. 🎉

**Pull request merged**